src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -16,19 +16,63 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// This class implements Test-Json command.
     /// </summary>
-    [Cmdlet(VerbsDiagnostic.Test, "Json", DefaultParameterSetName = ParameterAttribute.AllParameterSets, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096609")]
+    [Cmdlet(VerbsDiagnostic.Test, "Json", DefaultParameterSetName = JsonStringParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096609")]
     [OutputType(typeof(bool))]
     public class TestJsonCommand : PSCmdlet
     {
-        private const string SchemaFileParameterSet = "SchemaFile";
-        private const string SchemaStringParameterSet = "SchemaString";
+        #region Parameter Set Names
+
+        private const string JsonStringParameterSet = "JsonString";
+        private const string JsonStringWithSchemaStringParameterSet = "JsonStringWithSchemaString";
+        private const string JsonStringWithSchemaFileParameterSet = "JsonStringWithSchemaFile";
+        private const string JsonPathParameterSet = "JsonPath";
+        private const string JsonPathWithSchemaStringParameterSet = "JsonPathWithSchemaString";
+        private const string JsonPathWithSchemaFileParameterSet = "JsonPathWithSchemaFile";
```

Comment:
"JsonPath" is really unfortunate naming as it suggests JSON Path, the query syntax.

Reply:
@gregdennis Agree, we could change parameter set names to something more suitable.

Reply:
Please wait until my #18141 merges, or I can do it afterward.  That change almost erased these due to a bad rebase.